src/pages/errors/500/Page500.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Error from '../../../shared/components/error';
+
+const Page500 = (props) =>
+    (<Error
```

Comment:
Nifty pick: can we have the parenthesis in the first line?

Reply:
Sure, I also prefer that way, nice catch 👍